lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -2,6 +2,7 @@ defmodule Mix.Tasks.Escript.Install do
   use Mix.Task
 
   @shortdoc "Install an escript locally"
+  @preferred_cli_env :prod
```

Comment:
I think we should not do this and instead we should set Mix.env temporarily when building the other project only. :)